src/client/cli/cmd/launch.cpp

```diff
@@ -401,7 +367,7 @@ mp::ReturnCode cmd::Launch::request_launch(const ArgParser* parser)
         return ReturnCode::Ok;
     };
 
-    auto on_failure = [this](grpc::Status& status) {
+    auto on_failure = [this, &parser](grpc::Status& status, mp::LaunchReply& reply) {
```

Comment:
You need to pause the timer here, too?

```plain
$ multipass launch --network enx00133b4909cd --timeout 1 
Multipass needs to create a bridge to connect to enx00133b4909cd.
This will temporarily disrupt connectivity on that interface.

Timed out waiting for instance launch.
```

Reply:
Ahh yes, good catch!

Reply:
Done.